File: app/src/main/java/com/neome/api/home/drawer/msg/MsgLastMessageGet.java
// neome.ai API - do not change
//

package com.neome.api.home.drawer.msg;

import com.neome.api.core.base.msg.MsgVersion;
import com.neome.api.meta.base.Types.ContactId;

@SuppressWarnings("unused")
public class MsgLastMessageGet extends MsgVersion
{
  public ContactId chatId;
}

File: app/src/main/java/com/neome/api/home/drawer/msg/MsgLastMessageGet.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.home.drawer.msg

import com.neome.api.core.base.msg.MsgVersion
import com.neome.api.meta.base.Types.ContactId

open class MsgLastMessageGet : MsgVersion() {
    lateinit var chatId: ContactId
}
